### ISSUE TITLE
Binding Lua - Problem with global variable push

### ISSUE THREAD

**Issue by Jejeleterrible** [DigitalPulseSoftware/NazaraEngine#112]
When I want to use the `PushGlobal()` method of the *LuaInstance* with an *EntityHandle*, I must do 

``` cpp
lua.PushGlobal("Text", entity.GetObject()->CreateHandle());
```
instead of : 

``` cpp
lua.PushGlobal("Text", entity);
```

**Comment by Jejeleterrible** [comment]
Oh, and it's the same with all handles

**Comment by DPSLynix** [comment]
You only need to do `lua.PushGlobal("variable", entity->CreateHandle());`
But I agree it's not normal, since you have already have a handle, I think
it's because of the moving semantic.

I'll check that

**Comment by Jejeleterrible** [comment]
It works thanks a lot !

**Comment by DPSLynix** [comment]
Which compiler did you use? With MSVC14 the

```cpp
lua.PushGlobal("Text", entity);
```

juste works.

**Comment by DPSLynix** [comment]
This should be fixed with v0.2

**Comment by Jejeleterrible** [comment]
And it is !
